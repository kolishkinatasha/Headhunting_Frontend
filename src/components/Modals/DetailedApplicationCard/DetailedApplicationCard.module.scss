.title {
  width: 100%;
  margin-bottom: 10px;
}
.col {
  padding-left: 10px;
  padding-right: 10px;
  margin-bottom: 15px;
}
.input_title {
  margin-bottom: 10px;
  display: block;
}
.status {
  color: white;
  padding: 10px;
  width: fit-content;
  border-radius: 3px;
  margin-top: -10px;
}
.row_buttons {
  margin-bottom: 15px;
}

.btn_copy {
  margin: 30px;
}