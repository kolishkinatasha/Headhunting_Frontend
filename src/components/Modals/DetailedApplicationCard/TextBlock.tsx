import React from 'react';
import { Col, Typography } from '@ff/ui-kit';

import classes from './DetailedApplicationCard.module.scss';

type TextBlockProps = {
  title?: string;
  xs: number;
};
const TextBlock: React.FC<TextBlockProps> = ({ title, xs, children }) => (
  <>
    <Col xs={xs} className={classes.col}>
      <Typography node="div" className={classes.input_title} bold>
        {title}
      </Typography>
      {children}
    </Col>
  </>
);
export default TextBlock;
