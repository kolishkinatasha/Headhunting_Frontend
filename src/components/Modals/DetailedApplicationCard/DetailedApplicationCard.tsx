import React from 'react';
import { ModalBlockContent } from 'components/layouts/Modal';
import { observer } from 'mobx-react';
import { Link } from '@mui/material';
import applications from 'stores/application/applications';

import classes from './DetailedApplicationCard.module.scss';
import TextBlock from './TextBlock';
import profile from '../../../stores/profile/profile';

const DetailedApplicationCard: React.FC = observer(() => {
  const detailedApp = applications.filteredData[0];

  const allowRoles = ['Администратор', 'Главный рекрутер', 'Рекрутер'];

  const HighRoleInfoBlock: React.FC = observer(() => (
    <>
      <TextBlock xs={6} title="Приоритет">
        {detailedApp?.priority}
      </TextBlock>
      <TextBlock xs={6} title="Сложность">
        {detailedApp?.complexity}
      </TextBlock>
    </>
  ));

  const statusColor = () => {
    switch (detailedApp?.claimstatus) {
      case 'На рассмотрении':
      case 'На согласовании':
      case 'На исполнении':
      case 'Согласование с руководством':
      case 'Исполнена':
        return 'green';
      case 'Черновик':
      case 'Закрыта':
        return 'grey';
      case 'Отклонена':
      case 'Приостановлена':
        return 'red';
      default:
        return 'black';
    }
  };

  return (
    <>
      <ModalBlockContent
        title={`Заявка на подбор №${detailedApp?.id} от ${detailedApp?.creationdate}`}
      >
        <TextBlock xs={3}>
          <div
            className={classes.status}
            style={{ backgroundColor: statusColor() }}
          >
            {detailedApp?.claimstatus}
          </div>
        </TextBlock>
        <TextBlock xs={9}>
          <div>{detailedApp?.hr}</div>
        </TextBlock>
        <TextBlock xs={4} title="Автор заявки">
          <Link>{detailedApp?.autor}</Link>
        </TextBlock>
        <TextBlock xs={4} title="Руководитель / Аккаунт">
          <Link>{detailedApp?.confirmaccount}</Link>
        </TextBlock>
        <TextBlock xs={4} title="Рекрутер">
          <Link>{detailedApp?.hr}</Link>
          <p>Дата начала подбора {detailedApp?.begindate}</p>
        </TextBlock>
      </ModalBlockContent>
      <ModalBlockContent title="Информация по вакансии">
        <TextBlock xs={6} title="Вакансия штатного расписания">
          {detailedApp?.invacancy}
        </TextBlock>
        <TextBlock xs={6} title="Наименование вакансии для подбора">
          {detailedApp?.pubvacancy}
        </TextBlock>
        <TextBlock xs={6} title="Основание для подбора">
          {detailedApp?.reason}
        </TextBlock>
        <TextBlock xs={6} title="Обоснование">
          {detailedApp?.whyisitneeded}
        </TextBlock>
        <TextBlock xs={6} title="Характер работы">
          {detailedApp?.description}
        </TextBlock>
        <TextBlock xs={6} title="Локация">
          {detailedApp?.location}
        </TextBlock>
        <TextBlock xs={6} title="Аккаунт">
          {detailedApp?.confirmaccount}
        </TextBlock>
        <TextBlock xs={6} title="Департамент">
          {detailedApp?.department}
        </TextBlock>
        <TextBlock xs={3} title="Заработная плата">
          {detailedApp?.payment}
        </TextBlock>
        <TextBlock xs={3} title="Количество вакансий">
          {detailedApp?.numberofvacancies}
        </TextBlock>
        <TextBlock xs={6} title="Проект">
          {detailedApp?.project}
        </TextBlock>
      </ModalBlockContent>
      <ModalBlockContent title="Требования к компетентности сотрудника">
        <TextBlock xs={12} title="Образование">
          {detailedApp?.education}
        </TextBlock>
        <TextBlock xs={6} title="Основные навыки">
          {detailedApp?.basicskills}
        </TextBlock>
        <TextBlock xs={6} title="Дополнительные навыки">
          {detailedApp?.extraskills}
        </TextBlock>
        <TextBlock xs={6} title="Количество лет">
          {detailedApp?.numberofyears}
        </TextBlock>
        <TextBlock xs={6} title="Требуемый опыт">
          {detailedApp?.requiredexperience}
        </TextBlock>
        <TextBlock xs={12} title="Должностные обязанности">
          {detailedApp?.jobduties}
        </TextBlock>
        <TextBlock xs={5} title="График работы">
          {detailedApp?.schedule}
        </TextBlock>
        <TextBlock xs={3} title="Занятость">
          {detailedApp?.occupation}
        </TextBlock>
        <TextBlock xs={4} title="Испытательный срок (в месяцах)">
          {detailedApp?.probation}
        </TextBlock>
        <TextBlock xs={6} title="Наличие премирования">
          {detailedApp?.bonuses}
        </TextBlock>
      </ModalBlockContent>
      <ModalBlockContent title="Информация HR">
        {allowRoles.includes(profile.role) && <HighRoleInfoBlock />}
        <TextBlock xs={12} title="Рекрутер-исполнитель">
          {detailedApp?.hrpartner}
        </TextBlock>
        <TextBlock xs={6} title="Текущий статус для Аккаунта">
          {detailedApp?.claimstatus}
        </TextBlock>
        <TextBlock xs={6} title="Количество подобранных вакансий">
          {detailedApp?.numberofdonevacancies}
        </TextBlock>
        <TextBlock xs={6} title="ФИО кандидатов">
          {detailedApp?.applicantsfio}
        </TextBlock>
        <TextBlock xs={6} title="Дата выхода на работу" />
        <TextBlock xs={12} title="Проводящие собеседования">
          {detailedApp?.interviewers}
        </TextBlock>
      </ModalBlockContent>
    </>
  );
});
export default DetailedApplicationCard;
