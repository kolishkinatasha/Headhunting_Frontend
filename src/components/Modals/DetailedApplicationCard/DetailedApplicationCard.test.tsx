import React from 'react';
import { render } from '@testing-library/react';

import DetailedApplicationCard from './DetailedApplicationCard';

describe('DetailedApplicationCard', () => {
  it('should be the same', () => {
    const app = render(<DetailedApplicationCard />);
    expect(app).toMatchSnapshot();
  });
});
