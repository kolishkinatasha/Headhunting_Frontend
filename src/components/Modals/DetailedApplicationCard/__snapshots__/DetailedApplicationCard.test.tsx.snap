// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailedApplicationCard should be the same 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="block"
      >
        <h3
          class="fns-typography fns-typography_h3 title"
        >
          Заявка на подбор №1 от 22.10.2021
        </h3>
        <div
          class="fns-row block_content"
        >
          <div
            class="fns-col fns-col_xs-3 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            />
            <div
              class="status"
              style="background-color: green;"
            >
              На рассмотрении
            </div>
          </div>
          <div
            class="fns-col fns-col_xs-9 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            />
            <div>
              Иванов Иван HRович
            </div>
          </div>
          <div
            class="fns-col fns-col_xs-4 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Автор заявки
            </div>
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1iwtne7-MuiTypography-root-MuiLink-root"
            >
              Иванов Иван Авторович
            </a>
          </div>
          <div
            class="fns-col fns-col_xs-4 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Руководитель / Аккаунт
            </div>
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1iwtne7-MuiTypography-root-MuiLink-root"
            >
              Иванов Сергей Аккаунтович
            </a>
          </div>
          <div
            class="fns-col fns-col_xs-4 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Рекрутер
            </div>
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1iwtne7-MuiTypography-root-MuiLink-root"
            >
              Иванов Иван HRович
            </a>
            <p>
              Дата начала подбора 
              23.10.2021
            </p>
          </div>
        </div>
      </div>
      <div
        class="block"
      >
        <h3
          class="fns-typography fns-typography_h3 title"
        >
          Информация по вакансии
        </h3>
        <div
          class="fns-row block_content"
        >
          <div
            class="fns-col fns-col_xs-6 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Вакансия штатного расписания
            </div>
            web-разработчик React
          </div>
          <div
            class="fns-col fns-col_xs-6 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Наименование вакансии для подбора
            </div>
            Web-разработчик React
          </div>
          <div
            class="fns-col fns-col_xs-6 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Основание для подбора
            </div>
            Новый проект
          </div>
          <div
            class="fns-col fns-col_xs-6 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Обоснование
            </div>
            Доп текст по проекту
          </div>
          <div
            class="fns-col fns-col_xs-6 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Характер работы
            </div>
            Удаленно
          </div>
          <div
            class="fns-col fns-col_xs-6 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Локация
            </div>
            Москва
          </div>
          <div
            class="fns-col fns-col_xs-6 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Аккаунт
            </div>
            Иванов Сергей Аккаунтович
          </div>
          <div
            class="fns-col fns-col_xs-6 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Департамент
            </div>
            Департамент управления проектами
          </div>
          <div
            class="fns-col fns-col_xs-3 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Заработная плата
            </div>
            60000
          </div>
          <div
            class="fns-col fns-col_xs-3 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Количество вакансий
            </div>
            2
          </div>
          <div
            class="fns-col fns-col_xs-6 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Проект
            </div>
            HH
          </div>
        </div>
      </div>
      <div
        class="block"
      >
        <h3
          class="fns-typography fns-typography_h3 title"
        >
          Требования к компетентности сотрудника
        </h3>
        <div
          class="fns-row block_content"
        >
          <div
            class="fns-col fns-col_xs-12 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Образование
            </div>
            Высшее
          </div>
          <div
            class="fns-col fns-col_xs-6 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Основные навыки
            </div>
            Javascript
          </div>
          <div
            class="fns-col fns-col_xs-6 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Дополнительные навыки
            </div>
            Доп навыки тут
          </div>
          <div
            class="fns-col fns-col_xs-6 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Количество лет
            </div>
            1-3
          </div>
          <div
            class="fns-col fns-col_xs-6 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Требуемый опыт
            </div>
            Опыт разработки SPA
          </div>
          <div
            class="fns-col fns-col_xs-12 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Должностные обязанности
            </div>
            Красить кнопки
          </div>
          <div
            class="fns-col fns-col_xs-5 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              График работы
            </div>
            5/2
          </div>
          <div
            class="fns-col fns-col_xs-3 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Занятость
            </div>
            Полная
          </div>
          <div
            class="fns-col fns-col_xs-4 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Испытательный срок (в месяцах)
            </div>
            3
          </div>
          <div
            class="fns-col fns-col_xs-6 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Наличие премирования
            </div>
            Да
          </div>
        </div>
      </div>
      <div
        class="block"
      >
        <h3
          class="fns-typography fns-typography_h3 title"
        >
          Информация HR
        </h3>
        <div
          class="fns-row block_content"
        >
          <div
            class="fns-col fns-col_xs-6 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Приоритет
            </div>
            1
          </div>
          <div
            class="fns-col fns-col_xs-6 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Сложность
            </div>
            1
          </div>
          <div
            class="fns-col fns-col_xs-12 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Рекрутер-исполнитель
            </div>
            Иванов Иван Партнерович
          </div>
          <div
            class="fns-col fns-col_xs-6 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Текущий статус для Аккаунта
            </div>
            На рассмотрении
          </div>
          <div
            class="fns-col fns-col_xs-6 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Количество подобранных вакансий
            </div>
            2
          </div>
          <div
            class="fns-col fns-col_xs-6 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              ФИО кандидатов
            </div>
            Иванов Иван Заявович
          </div>
          <div
            class="fns-col fns-col_xs-6 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Дата выхода на работу
            </div>
          </div>
          <div
            class="fns-col fns-col_xs-12 col"
          >
            <div
              class="fns-typography fns-typography_bold input_title"
            >
              Проводящие собеседования
            </div>
            Иванов Иван Интервьюерович
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="block"
    >
      <h3
        class="fns-typography fns-typography_h3 title"
      >
        Заявка на подбор №1 от 22.10.2021
      </h3>
      <div
        class="fns-row block_content"
      >
        <div
          class="fns-col fns-col_xs-3 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          />
          <div
            class="status"
            style="background-color: green;"
          >
            На рассмотрении
          </div>
        </div>
        <div
          class="fns-col fns-col_xs-9 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          />
          <div>
            Иванов Иван HRович
          </div>
        </div>
        <div
          class="fns-col fns-col_xs-4 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Автор заявки
          </div>
          <a
            class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1iwtne7-MuiTypography-root-MuiLink-root"
          >
            Иванов Иван Авторович
          </a>
        </div>
        <div
          class="fns-col fns-col_xs-4 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Руководитель / Аккаунт
          </div>
          <a
            class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1iwtne7-MuiTypography-root-MuiLink-root"
          >
            Иванов Сергей Аккаунтович
          </a>
        </div>
        <div
          class="fns-col fns-col_xs-4 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Рекрутер
          </div>
          <a
            class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1iwtne7-MuiTypography-root-MuiLink-root"
          >
            Иванов Иван HRович
          </a>
          <p>
            Дата начала подбора 
            23.10.2021
          </p>
        </div>
      </div>
    </div>
    <div
      class="block"
    >
      <h3
        class="fns-typography fns-typography_h3 title"
      >
        Информация по вакансии
      </h3>
      <div
        class="fns-row block_content"
      >
        <div
          class="fns-col fns-col_xs-6 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Вакансия штатного расписания
          </div>
          web-разработчик React
        </div>
        <div
          class="fns-col fns-col_xs-6 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Наименование вакансии для подбора
          </div>
          Web-разработчик React
        </div>
        <div
          class="fns-col fns-col_xs-6 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Основание для подбора
          </div>
          Новый проект
        </div>
        <div
          class="fns-col fns-col_xs-6 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Обоснование
          </div>
          Доп текст по проекту
        </div>
        <div
          class="fns-col fns-col_xs-6 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Характер работы
          </div>
          Удаленно
        </div>
        <div
          class="fns-col fns-col_xs-6 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Локация
          </div>
          Москва
        </div>
        <div
          class="fns-col fns-col_xs-6 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Аккаунт
          </div>
          Иванов Сергей Аккаунтович
        </div>
        <div
          class="fns-col fns-col_xs-6 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Департамент
          </div>
          Департамент управления проектами
        </div>
        <div
          class="fns-col fns-col_xs-3 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Заработная плата
          </div>
          60000
        </div>
        <div
          class="fns-col fns-col_xs-3 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Количество вакансий
          </div>
          2
        </div>
        <div
          class="fns-col fns-col_xs-6 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Проект
          </div>
          HH
        </div>
      </div>
    </div>
    <div
      class="block"
    >
      <h3
        class="fns-typography fns-typography_h3 title"
      >
        Требования к компетентности сотрудника
      </h3>
      <div
        class="fns-row block_content"
      >
        <div
          class="fns-col fns-col_xs-12 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Образование
          </div>
          Высшее
        </div>
        <div
          class="fns-col fns-col_xs-6 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Основные навыки
          </div>
          Javascript
        </div>
        <div
          class="fns-col fns-col_xs-6 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Дополнительные навыки
          </div>
          Доп навыки тут
        </div>
        <div
          class="fns-col fns-col_xs-6 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Количество лет
          </div>
          1-3
        </div>
        <div
          class="fns-col fns-col_xs-6 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Требуемый опыт
          </div>
          Опыт разработки SPA
        </div>
        <div
          class="fns-col fns-col_xs-12 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Должностные обязанности
          </div>
          Красить кнопки
        </div>
        <div
          class="fns-col fns-col_xs-5 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            График работы
          </div>
          5/2
        </div>
        <div
          class="fns-col fns-col_xs-3 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Занятость
          </div>
          Полная
        </div>
        <div
          class="fns-col fns-col_xs-4 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Испытательный срок (в месяцах)
          </div>
          3
        </div>
        <div
          class="fns-col fns-col_xs-6 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Наличие премирования
          </div>
          Да
        </div>
      </div>
    </div>
    <div
      class="block"
    >
      <h3
        class="fns-typography fns-typography_h3 title"
      >
        Информация HR
      </h3>
      <div
        class="fns-row block_content"
      >
        <div
          class="fns-col fns-col_xs-6 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Приоритет
          </div>
          1
        </div>
        <div
          class="fns-col fns-col_xs-6 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Сложность
          </div>
          1
        </div>
        <div
          class="fns-col fns-col_xs-12 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Рекрутер-исполнитель
          </div>
          Иванов Иван Партнерович
        </div>
        <div
          class="fns-col fns-col_xs-6 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Текущий статус для Аккаунта
          </div>
          На рассмотрении
        </div>
        <div
          class="fns-col fns-col_xs-6 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Количество подобранных вакансий
          </div>
          2
        </div>
        <div
          class="fns-col fns-col_xs-6 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            ФИО кандидатов
          </div>
          Иванов Иван Заявович
        </div>
        <div
          class="fns-col fns-col_xs-6 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Дата выхода на работу
          </div>
        </div>
        <div
          class="fns-col fns-col_xs-12 col"
        >
          <div
            class="fns-typography fns-typography_bold input_title"
          >
            Проводящие собеседования
          </div>
          Иванов Иван Интервьюерович
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
